express-delete/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const newArray = [];
+  for (const objects in grades) {
+    newArray.push(grades[objects]);
+  }
+
+  res.json(newArray);
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  for (const objects in grades) {
+    if (req.params.id === objects) {
+      delete grades[objects];
+    }
+  }
```

Comment:
You can skip using a loop here, since you already have the id in `req.params.id`.